protocol/interoperable-ether.md

```diff
@@ -0,0 +1,198 @@
+# Purpose
+
+This document exists to align on a design for interoperable `ether`. Without this, user experience would
+be greatly hurt and `ether` is commonly used to pay for gas, which is necessary to use a chain.
+
+# Summary
+
+A new `WETH` predeploy is introduced that adds the `SuperchainERC20` functionality. It has a simple migration path from
+legacy `WETH` and maintains the ability to have interop with custom gas token chains.
+
+# Problem Statement + Context
+
+The enshrinement of native asset sends between chains was removed as part of the interop design. This greatly reduced
+the scope of the project as a native ether send feature could be subject to a double spend attack. Instead, we opt
+for using `WETH` to send between chains. The existing `WETH` contract is not proxied meaning that it cannot easily
+be upgraded. It also existed from before the times of `SuperchainERC20`, meaning that it has no built in cross chain
+sending ability.
+
+With the introduction of custom gas token, the existing `WETH` predeploy now has the semantics of "wrapped native asset",
+meaning that the `WETH` predeploy is not guaranteed to be `WETH`. This means one of two things:
+
+- A new `WETH` predeploy is introduced that supports `SuperchainERC20`
+- Custom gas token chains can never be interoperable
+
+# Proposed Solution
+
+A new `WETH` predeploy is introduced that supports interfaces `ISuperchainERC20`.
+This means that the new `WETH` predeploy allows for cross chain transfers.
+
+The main problem with this solution is the fact that it will break liquidity for wrapped `ether` into
+2 different contracts. Many applications already exist today that integrate with the existing predeploy.
+This will be annoying, but it should be very simple to migrate from the old `WETH` to the new `WETH`.
+Its a simple unwrap and wrap, and technically we could build support directly into the new `WETH` contract
```

Comment:
Relevant holders for OP Mainnet, looks like a mix of Aave and bridges: https://optimistic.etherscan.io/token/0x4200000000000000000000000000000000000006#balances